tests/conftest.py

```diff
@@ -134,6 +140,12 @@ def test_token_storage(mock_login_token_response):
         "auth_client_data",
         {"client_id": "fakeClientIDString", "client_secret": "fakeClientSecret"},
     )
+    # NB: this carefully matches the ID provided by our "foo_user_info" data fixture
+    # in the future, this should be moved to a dedicated fixture providing the current
+    # user's identity ID
+    mockstore.store_config(
+        "auth_user_data", {"sub": "25de0aed-aa83-4600-a1be-a62a910af116"}
+    )
```

Comment:
There's a broader refactor I'd like to do in order to fix this relationship with the fixture file data, but I'm intentionally putting that out of scope as this work already has so many touches on central parts of the application.